ingestion-beam/src/main/java/com/mozilla/telemetry/aet/DecryptAetIdentifiers.java

```diff
@@ -296,21 +228,19 @@ public PubsubMessage apply(PubsubMessage message) {
         throw new UnparsableAetPayloadException(e);
       }
 
-      // This transform comes early in the Decoder job before ParseUri, so we must work with
-      // the raw URI rather than parsed namespace and doctype.
-      // In particular, ParseUri is the first transform that can send messages to error output;
-      // we risk spilling sensitive information if AET payloads get sent to an output before anon_id
-      // values are removed, so we must have this transform come before an error step so that we
-      // can handle sanitizing/dropping the payload before emitting errors.
+      String namespace = message.getAttribute(Attribute.DOCUMENT_NAMESPACE);
+      String docType = message.getAttribute(Attribute.DOCUMENT_TYPE);
+
       try {
-        if (TELEMETRY_URI_PATTERN.matcher(uri).matches()) {
+        if (Namespace.TELEMETRY.equals(namespace) && //
+            docType != null && docType.startsWith("account-ecosystem")) {
```

Comment:
This could be moved into a `final String` variable, since it's used here and below.